CONTRIBUTING

```diff
@@ -0,0 +1,3 @@
+First off, thanks for taking the time to contribute!
+
+Either to [report a bug](http://ghdl-devfork.readthedocs.io/en/doc-gcc/contribute.html#reporting-bugs), [request enhancements](http://ghdl-devfork.readthedocs.io/en/doc-gcc/contribute.html#requesting-enhancements) or to [contribute modifications](http://ghdl-devfork.readthedocs.io/en/doc-gcc/contribute.html#fork-modify-and-pull-request), please read [Contributing](http://ghdl-devfork.readthedocs.io/en/doc-gcc/contribute.html) in the [Docs](http://ghdl-devfork.readthedocs.io/en/doc-gcc/index.html).
```

Comment:
It should be ghdl.readthedocs.io instead of ghdl-devfork

Reply:
Sure. Fixed.